src/app/page.tsx

```diff
@@ -25,39 +23,47 @@ export default function Home() {
     targetRes: "1000",
   });
 
+  // DISPLAY STATES
+
+  const [showSettings, setShowSettings] = useState<boolean>(false);
+  const [showProgress, setShowProgress] = useState<boolean>(false);
+  // to enable the progress set this to false
+  const [progressButton, setProgressButton] = useState<boolean>(true);
```

Comment:
I might be confused regarding this value you mentioned, but for me, _progressButton_ had to be set to false initially for the feature to work. As it is now (true), the "success" message pops up immediately upon clicking the _Generate HDR Image_ button, rather than the "images are being processed" message.

Maybe we should set this to false? (Or am I misunderstanding how this works?)